crates/noirc_evaluator/src/ssa_refactor/opt/flatten_cfg.rs

```diff
@@ -140,7 +140,12 @@ impl Ssa {
     /// For more information, see the module-level comment at the top of this file.
     pub(crate) fn flatten_cfg(mut self) -> Ssa {
         for function in self.functions.values_mut() {
-            flatten_function_cfg(function);
+            match function.runtime() {
+                RuntimeType::Acir => {
+                    flatten_function_cfg(function);
+                }
```

Comment:
Brillig doesn't want flattening?

Reply:
This could use a comment I think

Reply:
Brillig is generated before these steps are ran

Reply:
will add comments!

Reply:
Ideally this wouldn't be a possible value, but probably not practical to have a new IR for each pass :D